src/index.html

```diff
@@ -10,8 +10,168 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body autocomplete="off">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fieldset_info">
+        <legend>
+          Personal information
+        </legend>
+        <label class="label_Surname">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            class="marga"
+          >
+        </label>
+        <br>
```

Comment:
More correctly wrap each form field in block div element instead adding <br> and add this: 
![image](https://user-images.githubusercontent.com/54936044/226142805-db1397f4-b275-40d1-8246-b912b83c5776.png)
Please FIX for ALL cases